### PR TITLE
feat: Add `google` icon

### PR DESCRIPTION
**Pull request by fauxparse**
Add a Google icon for login using Google OAuth.

### CODE REVIEW

**Comment by wappsdotgr** [comment]
Since you included this google icon you can close mine in #93 or use it with a '+' for google plus

**Comment by thomasjulianstoelen** [comment]
hi, what is the hold up here?

**Comment by adaniello** [comment]
No way to merge this?

**Comment by locness3** [comment]
> @colebemis Since you included this google icon you can close mine in #93 or use it with a '+' for google plus

Google plus is closing

**Comment by locness3** [comment]
> No way to merge this?

@colebemis have to review first

**Comment by colebemis** [comment]
I'll take a look at this today 👍

**Comment by colebemis** [comment]
@fauxparse It looks like this icon is located in a `icons/logos` directory. Could you move into the `icons` directory?

File: icons/logos/google.svg
Comment (colebemis) [inline icons/logos/google.svg]: It looks like the formatting of this SVG file is slightly different than the rest of the files. Could you run `npm run build` to fix it? There are instructions on how to do that at the bottom of this comment: https://github.com/feathericons/feather/issues/171#issuecomment-455356985

Comment (fauxparse) [reply icons/logos/google.svg]: Fixed!

**Comment by fauxparse** [comment]
> @fauxparse It looks like this icon is located in a `icons/logos` directory. Could you move into the `icons` directory?

Done. I guess the `logos` directory existed when I made the pull request back in August... ;)

**Comment by fauxparse** [comment]
@colebemis Should be good to go now.

**Comment by johnletey** [comment]
What's the hold up on this?

**Comment by mxstbr** [comment]
Would _love_ to have this!

**Comment by tbouron** [comment]
Hey there, any reason this hasn't be merged yet? I too would love to have the Google logo for my Google AOuth button rather than relying on some PNG icon.

**Comment by locness3** [comment]
> Hey there, any reason this hasn't be merged yet?

Waiting for @colebemis ...

**Comment by mittalyashu** [comment]
There should be a minimum of 2px of spacing

image

**Comment by simonschaufi** [comment]
If I adjust the font awesome google icon to have just the line, then it looks like this: 
image

@fauxparse Could you adjust it to look similar like that?

**Comment by fauxparse** [comment]
> @fauxparse Could you adjust it to look similar like that?

This PR is over two years old, I'm really not sure I need to.

**Comment by NickeManarin** [comment]
@colebemis Could you take a look at this? This PR is quite old...

**Comment by omnibrain** [comment]
@colebemis please? Looks like all it needs at this point is one last approval. I think a lot of people are waiting for this

**Comment by locness3** [comment]
Sadly it looks like @colebemis doesn't want to merge new icons...

**Comment by omnibrain** [comment]
@colebemis I'll send you $5 in bitcoins if you just click that merge button. Please, oh please mighty maintainer, let us have this. We do not ask for much. We are but some humble developers, trying to make a living, barely getting by, craving that beautiful G. We never questioned your authority, never even as much as glanced  at the new font-awesome icons. Have mercy with us, o mighty maintainer.

**Comment by locness3** [comment]
@omnibrain not sure that's how things work. And what prevents you from downloading the svg from @fauxparse's fork or using his whole repo?

**Comment by omnibrain** [comment]
@locness3 using the official package is obviously nicer than using some fork but I guess that‘s what I‘ll do. I think what bothers me is that I don‘t see any reason for this not to be merged? Has feather icons reached the end of development? I‘m OK with any explanation, but no explanation at all is hard to accept.

**Comment by locness3** [comment]
@omnibrain Brand icons are deprecated in feather, read #763, tho it would be cool to merge the current brand icons requests and only reject further ones.
But more generally @colebemis doesn't give any sign of life concerning this project.

**Comment by NickeManarin** [comment]
> @omnibrain Brand icons are deprecated in feather

That's a harsh decision. Deprecating all brand icons instead of accepting the ones that are simple enough?
That's like deprecating all icons because life is complex and complicated. 
I'm sorry for mocking, but:

"Hur-dur, how can we draw _life_ in this simple manner if most things are so complex and can't be drawn in a 24x24px grid? Lets deprecate everything!"

**Comment by locness3** [comment]
The problem, i guess, is brand icons are unique, and not adaptable to any icon set.

And if all you need are brand icons SVGs, simpleicons.com dors the job.

**Comment by simonschaufi** [comment]
Everyone who wants to manually add the icon, here is the path from simpleicons a little bit adjusted:

```html
 <symbol id="google" viewBox="0 0 24 24">
    <path d="M12.219 10.43 12.219 14.199 18.457 14.199C18.207 15.816 16.574 18.941 12.219 18.941 8.465 18.941 5.402 15.836 5.402 12 5.402 8.164 8.469 5.059 12.219 5.059 14.355 5.059 15.785 5.965 16.605 6.754L19.59 3.875C17.672 2.086 15.188 1 12.219 1 6.137 1 1.219 5.918 1.219 12 1.219 18.082 6.137 23 12.219 23 18.57 23 22.781 18.535 22.781 12.25 22.781 11.527 22.703 10.977 22.605 10.43ZM12.219 10.43"/>
</symbol>
```

**Comment by emmasax4** [comment]
I'll echo @fauxparse @omnibrain and say that this would be soooo helpful!

@colebemis Please? 🙏🏼

**Comment by PeterShaggyNoble** [comment]
As per #763, brand icons have been deprecated here in favour of Simple Icons, where this is already available: https://simpleicons.org/?q=Google

[Google](https://simpleicons.org/?q=Google)

**Comment by emmasax4** [comment]
Ah yes, I saw that. I wasn't sure if that was the final decision yet or not. I found [this substitution](https://iconscout.com/icon/google-1767946) too, which sort of looks in the style of feather.

**Review by gaffy94 — APPROVED** [review]

**Review by gianfrancolombardo — APPROVED** [review]
Looks perfect

**Comment by vonec** [comment]
when will this get merged ?

**Comment by locness3** [comment]
Never #769

**Comment by locness3** [comment]
Never #763

**Pull request closed without merging**
